scripts/start.js

```diff
@@ -137,21 +142,19 @@ checkBrowsers(paths.appPath)
       openBrowser(urls.localUrlForBrowser);
     });
 
-    ['SIGINT', 'SIGTERM'].forEach(function (sig) {
-      process.on(sig, function () {
+    ['SIGINT', 'SIGTERM'].forEach(function(sig) {
+      process.on(sig, function() {
         devServer.close();
         process.exit();
       });
     });
 
     /**
      * Hook runElectronApp() to 'done' (compile) event
-     * 
+     *
      * Fails on error
      */
-    compiler.plugin('done',
-      stats => !stats.hasErrors() && runElectronApp()
-    );
+    compiler.plugin('done', stats => !stats.hasErrors() && runElectronApp(port));
```

Comment:
Ah, were we not forwarding the found port?! I wonder when that broke / if it just never worked o_O